commands/cluster_commands/install

```diff
@@ -90,9 +91,15 @@ if [ "${TARGET_PLATFORM}" = $CIVO ]; then
   # if a CIVO cluster was selected, pre-set the configuration
   CIVO_CLUSTER_ID=$(eval "$(getGumSpinnerOrLogger "fetching cluster id of ${CIVO_CLUSTER} cluster") civo k8s show ${CIVO_CLUSTER} -o custom -f ID")
 
-  GRAPPLE_DNS=${CIVO_CLUSTER}
+  if [ "${GRAPPLE_DNS}" == "" ]; then 
+      GRAPPLE_DNS=${CIVO_CLUSTER}
+  fi
+
+  # check if the GRAPPLE_DNS ends with .com to confirm a full domain-name was passed in GRAPPLE_DNS
```

Comment:
@zahid-iqbal-alpha 
I would rather go with something like that:
https://superuser.com/questions/750362/simplest-way-to-verify-if-a-given-string-is-a-valid-fqdn-name

not all domain names are ending with .com...